js/src/modals/CreateWallet/createWalletStore.js

```diff
@@ -125,24 +112,18 @@ export default class CreateWalletStore {
           key
         };
       })
-      .filter((step) => {
-        return (this.walletType !== 'WATCH' || step.key !== 'DEPLOYMENT');
-      });
+      .filter((step) => this.walletType === 'WATCH' || step.key !== 'INFO');
   }
 
-  @computed get waiting () {
-    this.steps
-      .map((s, idx) => ({ idx, waiting: s.waiting }))
-      .filter((s) => s.waiting)
-      .map((s) => s.idx);
-  }
+  constructor (api, props) {
```

Comment:
Would prefer being a bit more explicit here instead of just "props". Maybe the right approach would be to have

```
constructor (api, { accounts, onClose })
```

(Small difference, but always visible what is expected)